src/convenience/frameworks.ts

```diff
@@ -239,6 +243,7 @@ export type WorktopAdapter = (req: {
 
 /** AWS lambda serverless functions */
 const awsLambda: LambdaAdapter = (event, _context, callback) => ({
+    // TODO: add safe parse workaround
```

Comment:
Making it a function will btw fix this

Reply:
i just commited, but again late for 2 secs before you merged:)
```ts
function safeJsonParse(value: string, fallback = {} as any): any {
    try {
        return JSON.parse(value);
    } catch (_err) {
        return fallback;
    }
}
```

Reply:
Ah